Notifier.py

```diff
@@ -10,20 +11,44 @@ class Notifier:
         'Johannes': 'schmid.johannes90@gmail.com',
         'hannesxo': 'hannes322@yahoo.de'
     }
-
+    # Creating messages container
+    e_mail = MIMEMultipart('alternative')
```

Comment:
I would create the email object inside the `notify` method, since you want to actually create a new email for every user.
In this case this also works, but if you keep using the same object every time you could have that some of the fields are not overwritten (because maybe we don't want to use them in that moment) and then you have the old values still in there. Creating a new email from scratch avoids that.